### PR TITLE
[react-stripe-elements] - Allow id prop in ElementProps

### PR DESCRIPTION
**Pull request by thchia**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/stripe/react-stripe-elements/pull/178 (I made the changes in the original repo too).
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by thchia** [comment]
@santiagodoldan not sure who is the best person to @ here but you seem pretty active.

File: types/react-stripe-elements/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-// Type definitions for react-stripe-elements 1.0
+// Type definitions for react-stripe-elements 1.1
 // Project: https://github.com/stripe/react-stripe-elements#readme
 // Definitions by: dan-j <https://github.com/dan-j>
 //                 Santiago Doldan <https://github.com/santiagodoldan>
 //                 sonnysangha <https://github.com/sonnysangha>
 //                 Andrew Goh Yisheng <https://github.com/9y5>
+// 								 Thomas Chia <https://github.com/thchia>
```
Comment (santiagodoldan) [inline types/react-stripe-elements/index.d.ts]: Tabulate this correctly

**Review by santiagodoldan — CHANGES_REQUESTED** [review]

**Comment by santiagodoldan** [comment]
I'd merge both commits and add a better message, maybe mentioning the library you are modifying.

**Comment by thchia** [comment]
@santiagodoldan Thanks, have addressed the issues.

**Review by santiagodoldan — APPROVED** [review]

**Pull request merged**